std_msgs/Header header

## fleet info for robust payload control
float64  r_jx         ## [m]
float64  r_jy         ## [m]
float64  v_jx         ## [m/s]
float64  v_jy         ## [m/s]
float64  f_Ljx        ## [N]
float64  f_Ljy        ## [N]
float64  f_Ljz        ## [N]
float64  delta_jx     ## [N]
float64  delta_jy     ## [N]
float64  delta_jz     ## [N]
float64  rd_jx        ## [m]
float64  rd_jy        ## [m]
bool emergency        ##